src/www/ui_components/server_card/index.ts

```diff
@@ -17,6 +17,8 @@ import {LegacyElementMixin} from "@polymer/polymer/lib/legacy/legacy-element-mix
 
 import {ServerConnectionState} from "./server_connection_viz";
 
+export * from "./server_connection_viz";
```

Comment:
What's the motivation for this?

I find having a symbol be accessible in different modules confusing.